hstrat/_auxiliary_lib/_lazy_attach_stub.py

```diff
@@ -0,0 +1,39 @@
+from typing import Optional, Callable, Any
+
+from lazy_loader import attach_stub
+
+
+def lazy_attach_stub(
+    __name: str,
+    __file: str,
+    *,
+    launder: bool,
+    launder_names: Optional[list[str]] = None
+) -> tuple[Callable[[str], Any], Callable[[], list[str]], list[str]]:
+    """
+    An extension of the lazy_loader.attach_stub function.
+    __name and __file is __name__ and __file__ (see how to call
+    attach_stub: https://scientific-python.org/specs/spec-0001/).
+    This drastically reduces load times for the library.
+
+    If launder is True, the module name of an attr is set to the
+    __name argument (see ._launder_impl_modules). If launder_names is
+    given, then the attr is only launder if its name is in launder_names.
+
+    Returns what you should name __getattr__, __dir__, and __all__
```

Comment:
We should add See Also's (numpy style) to this and lazy_attach crossrefing eachother with a one sentence explanation of what the other does